practice_app/pages/api/getAllProducts.js

```diff
@@ -0,0 +1,46 @@
+const getAllProducts = (req, res) =>
+  new Promise(async (resolve) => {
+    // Imports the Google Cloud client library
+    const vision = require("@google-cloud/vision");
```

Comment:
It is bad practice to require inside functions, you can move any require to the top level (outside of the function)